doorway-ui-components/src/actions/Button.scss

```diff
@@ -187,3 +187,31 @@ button {
   transform: translate(-50%, -50%);
   margin-top: 2px;
 }
+
+// This is an override to update the style of the landing page buttons when selected.
+// This is not ideal but is needed since the landing page search buttons and the listing
+// search modal buttons have to have different styles when selected.
+.landing-search-button {
```

Comment:
supernit -- `landing-search-button` => `landing-search-button-group` since the class applies to the button group, not individual buttons and the selector is basically for `.buttons` within a `.landing-search-button-group`

Reply:
Done